JSDOMParser.js

```diff
@@ -0,0 +1,950 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Feels like this should go on its own node package eventually, and readability should have it as a dependency. Not now for sure ;)